cpu/esp32/usb_reset.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2023 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_esp32
+ * @{
+ *
+ * @file
+ * @brief       Trigger reset to the bootloader stored in the internal boot ROM
+ *              memory.
+ *
+ *              This will start the USB/UART bootloader.
+ *
+ * @author      Benjamin Valentin <benpicco@googlemail.com>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ *
+ * @}
+ */
+
+#include "driver/periph_ctrl.h"
+#include "rom/usb/chip_usb_dw_wrapper.h"
+#include "rom/usb/usb_persist.h"
+#include "soc/rtc_cntl_reg.h"
```

Comment:
```suggestion
#include "soc/soc_caps.h"
#include "soc/rtc_cntl_reg.h"
```
If you wanna test for `SOC_USB_SERIAL_JTAG_SUPPORTED`, `soc/soc_caps.h` has to be included.

Reply:
already [done](https://github.com/RIOT-OS/RIOT/compare/94b253fbb1a4322ecb4df0352a8c4ef135bafd01..ed5bfb7392cb5c4607b3066f57030f1b37af4a88)